types/web3/web3-tests.ts

```diff
@@ -1,4 +1,4 @@
-import Web3 = require("web3");
+import { Web3 } from "web3";
```

Comment:
This might work for the type definitions but not for the library which uses `module.exports`. Maybe this works when using synthetic default exports in the tsconfig.json but even if, then I would assume the correct syntax is actually `import Web3 from 'web3'`. Did you verify that this actually works when using `web3` in a project?